sources/iTermRootTerminalView.m

```diff
@@ -46,6 +47,14 @@ - (instancetype)initWithFrame:(NSRect)frameRect
     if (self) {
         _delegate = delegate;
 
+        self.visualEffectView = [[NSVisualEffectView alloc] initWithFrame:frameRect];
```

Comment:
I'd added manual release in `dealloc` method, because I assumed the `subview` will be a reference?

```objectivec
(void)dealloc {
   [_visualEffectView release];
```